pom.xml

```diff
@@ -56,11 +57,6 @@
 			<artifactId>spring-boot-starter-test</artifactId>
```

Comment:
Jeszcze pytanko - gdzie korzystasz z tej zależności?

Reply:
Z tej zależności korzystam, aż w jednym teście, który automatycznie wrzucany jest do zipa skręcanego za pomocą Spring Initializr'a (klasa TicketBookingApplicationTests).

Tutaj chciałabym się również odnieść do braku testów - mnie również boli. Planowałam testy, wiem że są one niezwykle ważne, stąd poniższa zależność już w pierwszym commicie:
```
<dependency>
	<groupId>org.assertj</groupId>
	<artifactId>assertj-core</artifactId>
	<scope>test</scope>
</dependency>
```
Później miałam jeszcze pomysł testowania endopointów przy pomocy MockMvc zamiast Postmana, ale zrezygnowałam. Dlaczego? Potrzebowałabym kolejnych kilku dni, a w międzyczasie zauważyłam, że ściągnęliście ogłoszenie ze swojej strony i Jooble - pomyślałam, że oferta jest już nieaktualna (brałam udział w rekrutacjach, które z powodu obecnej sytuacji zostały zawieszone, byłam i nie byłam o zawieszeniu rekrutacji informowana). A że zobowiązałam się do wykonania zadania, to mimo wszystko chciałam odesłać je w jak najszybszym stanie działającym i spełniającym wszystkie wymagania.
W najbliższym czasie planuję pracować nad testami, uczyć się najlepszego podejścia z TDD, i rozwijać tę umiejętność.
Przy okazji dziękuję za code review.